modules/wizard/wizard.php

```diff
@@ -36,6 +36,13 @@ class PLL_Wizard {
 	 */
 	protected $steps = array();
 
+	/**
+	 * The current step.
+	 *
+	 * @var string
+	 */
+	protected $step;
```

Comment:
I would say it's worth changing the variable's name in these cases. `$current_step` would help remind us what that variable holds throughout all the code base rather than needing to read it from the variable's declaration.

Reply:
It's not my goal to rewrite the code, as it's too many changes for my PR which only focuses on declarations and doc.